package-lock.json

```
@@ -14,7 +14,7 @@
         "@fortawesome/free-regular-svg-icons": "6.4.0",
         "@fortawesome/free-solid-svg-icons": "6.4.0",
         "@fortawesome/react-native-fontawesome": "0.2.7",
-        "@hathor/hathor-rpc-handler": "^0.0.1-experimental-alpha",
+        "@hathor/hathor-rpc-handler": "^0.0.2-experimental-alpha",
```

suggestion(non-blocking): While we're on the experimental phase, maybe it would be better not to use any automatic updates, as they may have breaking changes.
```suggestion
        "@hathor/hathor-rpc-handler": "0.0.2-experimental-alpha",
```